types/jshint/index.d.ts

```diff
@@ -0,0 +1,80 @@
+// Type definitions for jshint 2.12
+// Project: http://jshint.com/
```

Comment:
```suggestion
// Project: https://github.com/jshint/jshint
```
this is optional, can we swap this to repo location? thx!